src/runtime/finalizer.cs

```diff
@@ -75,19 +72,17 @@ private Finalizer()
             Threshold = 200;
         }
 
-        public void Collect(bool forceDispose = true)
+        [Obsolete("forceDispose parameter is unused. All objects are disposed regardless.")]
+        public void Collect(bool forceDispose) => this.DisposeAll();
+        public void Collect() => this.DisposeAll();
+
+        int _throttled;
```

Comment:
Is it better to move it to the fields region than isolate it on here?

Reply:
1. There's no fields region in this class.
2. This field is not used outside of the `ThrottledCollect` function.

Reply:
I mean put into here for example.
```
         private ConcurrentQueue<IPyDisposable> _objQueue = new ConcurrentQueue<IPyDisposable>();
         private int _throttled;
```